plugins/locator/stdloc/descriptions/global_stdloc.xml

```diff
@@ -17,7 +17,7 @@
 				<group name="profile">
 					<struct type="stdloc profile" link = "StdLoc.profiles">
 
-						<parameter name="method" type="string" default="GridSearch+LeastSquares">
+						<parameter name="method" type="string" default="LeastSquares">
 							<description>The location method to use: LeastSquares, GridSearch, OctTree,
 							 GridSearch+LeastSquares or OctTree+LeastSquares.
```

Comment:
Please use the "values" tag here, values="...,...". This is evaluated in scconfig and the HTML documentation and may be supported by pull-down menus in the future.